render-ws-spark-client/src/main/java/org/janelia/render/client/spark/pipeline/AlignmentPipelineParameters.java

```diff
@@ -63,62 +73,48 @@ public MultiProjectParameters getMultiProject() {
         return multiProject;
     }
 
-    public boolean hasMipmapParameters() {
-        return mipmap != null;
-    }
-
```

Comment:
Why did you delete all those `has...` methods? Just asking to understand.
Edit: saw that these were only used in `AlignmentPipelineClient` to check if the pipeline step is to be executed. I like the current design much more!